### ISSUE TITLE
image paths and previews not viewing

### ISSUE THREAD

**Issue by varndellwagglebee** [sveltia/sveltia-cms#222]
Images are not being saved with the correct path.

It's saving as 
image:/food5.jpg instead of images/uploads/food5.jpg

Here is my config:
```
media_folder: "/images/uploads/" # Folder where user uploaded files should go  
public_folder: "/"

collections: # A list of collections the CMS should be able to edit
  - name: "post" # Used in routes, ie.: /admin/collections/:slug/edit
    label: "Post" # Used in the UI, ie.: "New Post"
    folder: "_posts" # The path to the folder where the documents are stored
    sort: "date:desc" # Default is title:asc
    create: true # Allow users to create new documents in this collection
    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
    icon: post_add
    fields: # The fields each document in this collection have
      - { label: "Layout", name: "layout", widget: "hidden", default: "post" }
      - { label: "Title", name: "title", widget: "string", tagname: "h1" }
      - { label: "Body", name: "body", widget: "markdown" }
      - {
          label: "Publish Date",
          name: "date",
          widget: "datetime",
          format: "YYYY-MM-DD hh:mm:ss",
        }
      - { label: "Image", name: "image", widget: "image", required: false }
      - { label: "Tags", name: "tags", widget: "string", required: false }
```

**Comment by kyoshino** [comment]
Will check.

**Comment by varndellwagglebee** [comment]
Here is what the path is when I select in admin
![image](https://github.com/user-attachments/assets/860e35be-670f-4c8a-bf68-040ac1bfac99)

**Comment by kyoshino** [comment]
No, this is expected.

- `media_folder` is an internal path on the repository
- `public_folder is a public path part of the URL

and the `image` field uses `public_folder . Please refer to the [document](https://decapcms.org/docs/configuration-options/#media-and-public-folders).

**Comment by kyoshino** [comment]
I just checked [your repository](https://github.com/Stillpoint-Software/GoodThingsFromScratch). I think your configuration should be:

```yaml
media_folder: "images/uploads"
public_folder: "/images/uploads"
```

**Comment by varndellwagglebee** [comment]
I updated the folder paths and it's still not working.

![image](https://github.com/user-attachments/assets/7cfec595-ed57-47e1-9bdb-6fab59d50f5e)
![preview](https://github.com/user-attachments/assets/c05e19c1-e895-417d-8975-9941f56053f9)

**Comment by kyoshino** [comment]
Let me clone your repo and test.

**Comment by kyoshino** [comment]
It’s indeed a bug in Sveltia CMS. Not sure why I haven’t noticed this earlier. Fix incoming shortly.

**Comment by varndellwagglebee** [comment]
Thank you for looking at this so promptly.

**Comment by kyoshino** [comment]
Fixed in [v0.40.1](https://github.com/sveltia/sveltia-cms/releases/tag/v0.40.1).
